### ISSUE TITLE
Add mouse wheel sensitivity adjustment to mouse configuration

### ISSUE THREAD

**Issue by CodehunterWorks** [linuxmint/cinnamon#9815]
* Cinnamon version (4.6.7)
 * Distribution - (Mint 20 x64)
 * Graphics hardware *and* driver used (Intel Corporation Core Processor Integrated Graphics Controller 5.4.0-58-generic i915)
 ```

Please add an global configuration option in Cinnamon mouse settings to adjust the mouse wheel sensitivity like Win10 has:

![grafik](https://user-images.githubusercontent.com/35778828/103439032-b21ca280-4c39-11eb-95af-976aa5e3d133.png)

Many thanks!
-Cody

**Comment by IvoAndonov** [comment]
There already IS such an option! I made a screenshot of my Mouse Settings and added translation to the relevant sections bc my interface is in Cyrillic.

![Picture_created_04-01-2021_00-06-15](https://user-images.githubusercontent.com/41172201/103489958-2cf3e200-4e21-11eb-92d3-dac576c43fb6.png)

**Comment by CodehunterWorks** [comment]
Hi IvoAndonov,

you are wrong, unfortunately. None of these is related to Mouse **Wheel**. Size = Cursor size; Custom Accelleration = Cursor speed; Custom sensitivity = mostly used for Touchpad sensitivity. What I mean is the Step-Width (or Line-Count) which is scrolled when the Mousewheel is moved one "tick".

Greets
Cody

**Comment by IvoAndonov** [comment]
You're right. My bad. I thought you were talking about the cursor sensitivity.
I've never felt the need to change the wheel sensitivity since it has always been 5 lines and I like it that way (that's number I used to use when I was a Windows user).
If you're trying to have a better scrolling speed in the browser, maybe I can help you there, if it's Firefox or Firefox-based? Firefox for Linux ignores the default scroll values, IDK why.

**Comment by hedi-sel** [comment]
I agree. I've been seeing people complain about this for a while.
If you are looking for a workaround, [here it is](https://askubuntu.com/questions/285689/increase-mouse-wheel-scroll-speed)

**Comment by CodehunterWorks** [comment]
> I agree. I've been seeing people complain about this for a while.
> If you are looking for a simple workaround, [here it is](https://askubuntu.com/questions/285689/increase-mouse-wheel-scroll-speed)

I'm with you (imwheel). But I think, a UI in the system settings will increase the usability. Cinnamon is one of the most user friedly desktops for former Windows users. Such configuration options will close the gap a little bit.

About another suggestion in your linked post: I find it a very cheap solution to say "if your mouse wheel doesnt fit to the software behavior, buy another mouse". Remember how often we say "if your machine is to slow to run Win 10, use Linux!" ;-)

**Comment by Butterfly1984** [comment]
Mint 20.2, fully updated, Kernel 4.11

I just switched to a different mouse today, and the scroll speed is quite a bit lower than my previous mouse.
I'd certainly want an option in the "Mouse and Touchpad" settings menu to set up scroll speed/scroll lines.

**Comment by Pingger** [comment]
> There already IS such an option! I made a screenshot of my Mouse Settings and added translation to the relevant sections bc my interface is in Cyrillic.
> 
> ![Picture_created_04-01-2021_00-06-15](https://user-images.githubusercontent.com/41172201/103489958-2cf3e200-4e21-11eb-92d3-dac576c43fb6.png)

just an FYI: At least for me these two settings do exactly nothing! No matter how I configure the cursor speed stays the same.

**Comment by CodehunterWorks** [comment]
Its simple: When you have an notebook with touchpad, these settings are irrelevant to you. Same on the second tab, when you don't have an touchpad, they are senseless for you. But none of them are related to the mouse wheel (or the virtual wheel track on an touchpad).

**Comment by Pingger** [comment]
Let me rephrase that: The sensitivity and acceleration sliders and toggles on both pages (Mouse and Touchpad) don't actually do anything. checking stacktraces through gdb even shows, that only gui updates are triggered and no setting is actually applied anywhere. The cursor-speed and -acceleration for both mouse and touchpad have exactly zero functionality on quite a lot of systems as can be seen by quite a few of issues on this topic

**Comment by CodehunterWorks** [comment]
Hmmm this is possibly an driver issue to your hardware. On my notebook, both of these settings does work properly and instantly when its changed.

**Comment by AtomicRobotMan0101** [comment]
see here: https://github.com/linuxmint/cinnamon/issues/10480#issuecomment-1312512287

Botttom of the QT5 Configuration Tool....

![mouse scroll](https://user-images.githubusercontent.com/64177332/201482461-47cd26aa-cd96-4102-b26f-6d7e1ec9de2c.png)

**Comment by CodehunterWorks** [comment]
> see here: [#10480 (comment)](https://github.com/linuxmint/cinnamon/issues/10480#issuecomment-1312512287)
> 
> Botttom of the QT5 Configuration Tool....

Damn... Youre right. It feels like the configuration pain in Win8-10-11: A bunch of config tools for the same thing.

**Comment by Pingger** [comment]
still open.

**Comment by JosephMcc** [comment]
All feature requests now belong here: https://github.com/orgs/linuxmint/discussions
